indica/operators/atomic_data.py

```diff
@@ -536,6 +535,11 @@ def __call__(  # type: ignore
 
             self.F_z_t = F_z_t
         else:
+
+            # TODO: FIX THIS (Nh is ignored!?!)
+            # Not too mention why does F_z_t have extra dims
+            # (which are dropped..) other than Te / Ne / Nh???
```

Comment:
I think I understand this F_z_t a bit more... As it is know Nh is fixed at each radial point which isn't great,

Reply:
I agree, but it's a good approximation. Better a rough Nh than none. 
We had previously discussed a tabulation of Fz on a Te, Nh matrix, but this is an overkill at the moment, since we're fixing Te and Ne using TS and Nh is an informed (wild) guess.